<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SniffWindow</class>
 <widget class="QMainWindow" name="SniffWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="layout_filter">
        <item>
         <widget class="QLabel" name="lb_filter">
          <property name="text">
           <string>F&amp;ilter:</string>
          </property>
          <property name="buddy">
           <cstring>tb_filter</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="tb_filter"/>
        </item>
        <item>
         <widget class="QPushButton" name="bt_filter_clear">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_filter_apply">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>&amp;Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QTableWidget" name="table_packets">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
        <widget class="QTreeWidget" name="tree_packet">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
        <widget class="QHexEdit" name="hexEdit">
         <property name="toolTip">
          <string>A Simple Hex Editor</string>
         </property>
         <property name="whatsThis">
          <string>QHexEdit is a simple Hexeditor.   You can show end edit binary data.  </string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Sans</family>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menu_Sniff">
    <property name="title">
     <string>&amp;Sniff</string>
    </property>
    <addaction name="actionStop"/>
    <addaction name="actionSniff"/>
   </widget>
   <widget class="QMenu" name="menuStats">
    <property name="title">
     <string>S&amp;tats</string>
    </property>
    <addaction name="actionTable"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Sniff"/>
   <addaction name="menuAbout"/>
   <addaction name="menuStats"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave_As"/>
   <addaction name="actionSave"/>
   <addaction name="actionOpen"/>
   <addaction name="actionQuit"/>
   <addaction name="separator"/>
   <addaction name="actionSniff"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/save-as.png</normaloff>:/icons/save-as.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save As</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/capture_stop.png</normaloff>:/icons/capture_stop.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Stop</string>
   </property>
  </action>
  <action name="actionSniff">
   <property name="icon">
    <iconset resource="icons/icons.qrc">
     <normaloff>:/icons/capture_start.png</normaloff>:/icons/capture_start.png</iconset>
   </property>
   <property name="text">
    <string>Sni&amp;ff</string>
   </property>
  </action>
  <action name="actionTable">
   <property name="text">
    <string>Table</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QHexEdit</class>
   <extends>QScrollArea</extends>
   <header>QHexEdit/qhexedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>SniffWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
